OpenSim/Simulation/Model/AbstractTool.cpp

```diff
@@ -617,11 +617,11 @@ bool AbstractTool::createExternalLoads( const string& aExternalLoadsFileName, Mo
         // Form complete storage so that the kinematics match the state labels/ordering
         Storage *qStore=NULL;
         Storage *uStore=NULL;
-        aModel.getSimbodyEngine().formCompleteStorages(s, *loadKinematicsForPointTransformation,qStore,uStore);
+        aModel.getSimbodyEngine().formCompleteStorages(s, 
+            *loadKinematicsForPointTransformation,
+            qStore,
+            uStore);
```

Comment:
With the removal of `if (qStore->isInDegrees())`, it looks like a flag is now being ignored. The comment `qStore should be in radians` implies that the units of `qStore` must be checked ("should be"). Something like the following may be appropriate:
```cpp
// qStore is always in radians
if (qStore->isInDegrees()) { qStore->setIsInDegrees(false); }
```
…but that doesn't seem like something the consumer should be doing. Instead, perhaps throw an exception if `qStore->isInDegrees()` (i.e., `qStore` has unexpected property)?

Reply:
I should have removed the comment. `qStore` is in radians since `formCompleteStorages` by definition now only returns rotational coordinates in radians and radians/s.

I will remove it now.